test/CadlRanchProjects/type/property/nullable/src/Generated/Models/CollectionsByteProperty.cs

```diff
@@ -18,14 +18,13 @@ public partial class CollectionsByteProperty
         /// <summary> Initializes a new instance of CollectionsByteProperty. </summary>
         /// <param name="requiredProperty"> Required property. </param>
         /// <param name="nullableProperty"> Property. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="requiredProperty"/> or <paramref name="nullableProperty"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="requiredProperty"/> is null. </exception>
         internal CollectionsByteProperty(string requiredProperty, IEnumerable<BinaryData> nullableProperty)
```

Comment:
This corresponds to the typespec:
```
model CollectionsByteProperty {
nullableProperty: bytes[] | null;
}
```
But in our design, an array should never be null. Should we change the behavior here back?

Reply:
Good catch let me get some more information on this.